src/geometry/euclidean/angle/oriented/basic.lean

```diff
@@ -564,6 +564,11 @@ begin
   abel,
 end
 
+/-- Rotating twice is equivalent to rotating by the sum of the angles. -/
+@[simp] lemma rotation_apply_apply (θ₁ θ₂ : real.angle) (x : V) :
```

Comment:
I think `rotation_rotation` would be a better name here.

Note also that the proof would be one line shorter if you proved this first, and then proved `rotation_trans` in terms of it; right now this theorem statement is exactly the goal state on line 559.